developer/js/build.sh

```diff
@@ -11,7 +11,7 @@ set -eu
 EX_USAGE=64
 
 # Where to find lexical model types.
-LEXICAL_MODELS_TYPES=../../common/lexical-model-types
+LEXICAL_MODELS_TYPES=../../common/models/types
```

Comment:
So @jahorton, `developer/js` is not a Lerna-managed package? Why not?

Reply:
Because https://github.com/keymanapp/keyman/pull/3071/ isn't approved/merged yet.